assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -115,7 +115,9 @@ function useUploadMedia({ media, pagingNum, mediaType, fetchMedia, setMedia }) {
         fetchMedia({ pagingNum, mediaType }, setMedia);
       } catch (e) {
         showSnackbar({
-          message: e.message,
+          message: createInterpolateElement(e.message, {
```

Comment:
What if `e.message` doesn't have any `<b>` tag in it? Does it break?

Reply:
Why would it?

Reply:
It was broken as createInterpolateElement was not being imported from wp-elements anymore for some reason.I have copied the file over to make this work.

Reply:
createInterpolateElement has been removed.